server/src/app/config.py

```diff
@@ -4,8 +4,23 @@
 # See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
 
 
+import json
+
 PROJECT_NAME: str = 'Pyronear local webserver'
 PROJECT_DESCRIPTION: str = 'API for pyronear systems interactions on sites'
 API_BASE: str = 'api/'
 VERSION: str = "0.1.0a0"
 LOGO_URL: str = "https://github.com/pyronear/PyroNear/raw/master/docs/source/_static/img/pyronear-logo-dark.png"
+
+
+# security
+JWT_ENCODING_ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 365 * 10
+
+# to get a string like this run:
+# openssl rand -hex 32
+SECRET_KEY = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7"
```

Comment:
Time to get lectured :sweat_smile: 
**We NEVER EVER EVER EVER EVER put a secret key in a commit / PR / etc**. Even if you delete it in your next commit, the PR history will keep it.

So :
- from a security perspective, let's never do this again
- this secret key is compromised, so we will need to generate another.
- Github Secrets and `.env` files are your friends ;)

Reply:
As long this key is the one from the fastapi tutorial it was not planned to keep this one anyway ! 
Thought it would be easier to test/debug but should have had a debug key mode  and put aBother key in secrets Shard with pyro users